style.css

```
@@ -0,0 +1,117 @@
+:root {
+  --basic-white: #ffffff;
+  --basic-blue: #81b3d2;
+  --basic-lightgray: #eeeeee;
+  --basic-brown: #766357;
+  --basic-black: #000000;
+}
+body {
+  min-width: 960px;
+  margin: 0;
+  padding: 0;
+  font-family: "PT Sans", Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  color: var(--basic-black);
+  text-transform: uppercase;
+  background-color: var(--basic-white);
+  font-weight: 400;
+}
+a {
+  text-decoration: none;
+}
+img {
+  max-width: 100%;
+  height: auto;
+}
+.main-navigation {
+  font-size: 14px;
+  line-height: 26px;
+  color: var(--basic-black);
+  background-color: transparent;
+}
+.site-navigation {
+  list-style: none;
+}
+.site-navigation {
+  color: var(--basic-black);
+}
```

Зачем 2 селектора одинаковых?
```suggestion
.site-navigation {
  list-style: none;
  color: var(--basic-black);
}
```